src/index.js

```diff
@@ -20,15 +20,77 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(viewNumber, operator, inputNumber, resultNumber) {
```

Comment:
```suggestion
function render({ viewNumber, operator, inputNumber, resultNumber} ) {
```

자바스크립트는 매개변수도 destructuring 이 가능해요.
예를들어 `render(i, '', i, 0);` 라는 코드를 읽을 때, 첫번 째가 뭐였더라, 세 번째는 뭐였지라고 다시 확인하게 돼요.
그런데 destructuring 으로 표현해주면

```javascript
render({
  viewNumber: i,
  operator: '',
  inputNumber: i,
  resultNumber: 0,
})
```
처럼 표현도 명확해지고, 매개변수를 넣어줄 때 실수할 확률도 줄여줄 수 있어요.

Reply:
네~ 감사드립니다. 
Destructurng 참고 자료를 통해 공부햅고 적용해보겠습니다 🙏